intelmq/bots/parsers/alienvault/parser-otx.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import sys
+import json
+
+from intelmq.lib import utils
+from intelmq.lib.bot import Bot
+from intelmq.lib.harmonization import DateTime
+from intelmq.lib.message import Event
+
+HASHES = {
+    'FileHash-SHA256': 'SHA-256',
+    'FileHash-SHA1': 'SHA-1',
+    'FileHash-MD5': 'MD5'
+}
+
+
+class AlienVaultOTXParserBot(Bot):
+
+    def process(self):
+        report = self.receive_message()
+        if (report is None or not report.contains("raw") or
+                len(report.value("raw").strip()) == 0):
```

Comment:
An empty raw (after strip) will be rejected anyway: https://github.com/certtools/intelmq/blob/master/intelmq/lib/harmonization.py#L59